### ISSUE TITLE
[A11y]Status.NuGet.Org must not use meta refresh

### ISSUE THREAD

**Issue by skofman1** [NuGet/NuGetGallery#7855]
@ryuyu commented on [Thu Feb 27 2020](https://github.com/NuGet/Engineering/issues/3018)

Currently, the status page uses a <meta> tag to cause the page to refresh.

This is not a11y compliant.
The page should be updated to use a javascript refresh instead.
